src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Transformers/ManagementHierarchyTransformer.cs

```diff
@@ -0,0 +1,105 @@
+﻿using SFA.DAS.RoatpAssessor.Web.ApplyTypes.Apply;
```

Comment:
Test coverage for this class?

Reply:
Added 8 test cases to provide requested coverage